Jenkinsfile

```
@@ -118,6 +118,34 @@ pipeline {
             defaultValue: false,
             description: 'Check this box to setup Kafka for NetObserv'
         )
+        separator(
+            name: 'KAFKA_CONFIG_OPTIONS',
+            sectionHeader: 'KAFKA Configuration Options',
+            sectionHeaderStyle: '''
+                font-size: 14px;
+                font-weight: bold;
+                font-family: 'Orienta', sans-serif;
+            '''
+        )
+        choice(
+            name: 'TOPIC_PARTITIONS',
+            choices: [6, 10, 24, 48],
+            description: '''
+            Number of Kafka Topic Partition. Also, adjust number of FLP Replicas. Below are recommended values for paritions:<br/>
+            6 - default for non-perf testing environments<br/>
```

```suggestion
            6 - default for non-perf testing environments<br/>
```
In what scenarios would we use this instead of 10?

---

Fine with me, just looking for an explanation, thanks!

---

One further question, you say FLP replicas are set here in the desc, but your answer makes it sound like they are set in the `TOPIC_PARTITIONS` field, which is it?

---

ah, it is `TOPIC_PARTITIONS`. That text is remnant of my initial thought on implementation where I wanted to scale FLP replicas automatically based on TOPIC_PARTITIONS value, but having separate `FLP_KAFKA_REPLICAS` as in this PR gives user more flexibility. I'll remove `Also, adjust number of FLP Replicas ` text.